_sass/_basic.scss

```diff
@@ -47,39 +47,6 @@
     border-top: 1px solid $teachla-green;
 }
 
-.row {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    width: 100%;
-    height: auto;
-    margin-top: 1em;
-    margin-bottom: 1em;
-}
-.col{
-    max-width: 100%;
-    height: 100%;
-    @media (min-width: 700px) {
-        display: flex;
-        flex-direction: column;
-        flex-basis: 100%;
-        flex: 1;
-        padding-left: 1em;
-        padding-right: 1em;
-    }
-}
```

Comment:
Unfortunately you can't delete these classes because other pages on the site rely on them!